test/karma/main.test.js

```diff
@@ -63,6 +64,27 @@ describe('Root', function () {
     expect(TestUtils.isDOMComponent(login)).to.be.true
   })
 
+  it('should render the cookie banner if the cookie is not set', () => {
+    CookieBanner.deleteCookie()
+
+    const {root} = renderRootWithInitialState({
+      auth: {isAuthenticated: false}
+    })
+    const cookieBanner = TestUtils.findRenderedComponentWithType(root, CookieBanner)
+    expect(TestUtils.isCompositeComponent(cookieBanner)).to.be.true
+  })
+
+  it.skip('should not render the cookie banner if the cookie is set', () => {
```

Comment:
No, the test doesn't work right now, because either the cookie is not recognized in the Karma environment or because the assertion is somehow wrong. Could also be a rendering timing issue, not sure.